### PR TITLE
Add nitay880 to contributors route

### PR DESCRIPTION
**Pull request by Nitay880**
Please add me :)

### CODE REVIEW

File: backend/tests/test_routes.py [hunk 1]
```diff
@@ -91,6 +91,5 @@ def test_get_contributors(client):
     result = response.json()
     assert len(result) == 1
     assert result == [{
-        'name': 'nvg',
-        'github_profile': 'https://github.com/nvgoldin'
-    }]
+        'name': 'nitai_m',
+        'github_profile': 'https://github.com/Nitay880'}]
```
Comment (nvgoldin) [inline backend/tests/test_routes.py]: don't remove, add another one! :)

**Review by nvgoldin — CHANGES_REQUESTED** [review]

File: backend/tests/test_routes.py (hunk 1 above)
Comment (Nitay880) [reply backend/tests/test_routes.py]: done

File: backend/zulu/db_tools.py [hunk 2]
```diff
@@ -1,15 +1,20 @@
...
+ # a user references a story which references a point
 def points_db():
     client = MongoClient(settings.MONGO_URI).geo_data
     collection = client.points
-
     collection.create_index([("location", GEOSPHERE)])
     yield collection
-
-
+def story_db():
```
Comment (Nitay880) [inline backend/zulu/db_tools.py:10]: well maybe stories need a DB for themselves, 
a goot arch will possibly be
user DB, points DB, stories DB
user ----> [bunch of points] 
point ----->[bunch of stories]

**Comment by matanel-oren** [comment]
not relevant anymore

**Pull request closed without merging**